### ISSUE TITLE
Aliases aren't being followed (opens Finder instead)

### ISSUE THREAD

**Issue by Amparose** [marta-file-manager/marta-issues#845]
This is annoying and seems like unintentional behaviour.

**Comment by CodeGrammer45** [comment]
Yes, I was looking in the settings, and I haven't been able to find anything that can fix this behavior, but it is pretty annoying to have to right-click and choose Open With > Navigate to Original File...

I think this should be a setting that we can set.

Edit: I just found this from 2017:
https://github.com/marta-file-manager/marta-issues/issues/72#issue-243184847
Seems like it used to work the way I wish it did now lol

**Comment by royemosby** [comment]
Just ran into this too. I have things aliased all over the place just so I can navigate flexibly.

**Comment by royemosby** [comment]
To follow up, "Navigate to the original alias" works but I have a bad memory and I would just expect to be able to follow it the same way as a folder

**Comment by fdawans** [comment]
I would like to be able to quickly access favorites in a single click like in Finder.
So I added a folder with aliases and used it as default folder as a fix - but it opens Finder instead...
Maybe there is an alternative way to do this?
